lib/plug/static.ex

```diff
@@ -179,10 +185,11 @@ defmodule Plug.Static do
 
   defp serve_static({:ok, conn, file_info, path}, segments, options) do
     %{qs_cache: qs_cache, et_cache: et_cache, et_generation: et_generation,
-      gzip?: gzip?, brotli?: brotli?, headers: headers} = options
+      gzip?: gzip?, brotli?: brotli?, headers: headers, types: types} = options
     case put_cache_header(conn, qs_cache, et_cache, et_generation, file_info, path) do
       {:stale, conn} ->
-        content_type = segments |> List.last |> MIME.from_path
+        fname = List.last(segments)
```

Comment:
Yes, you think it's not clear enough? I usually don't use short var name, but as it's used only on the next line I used fname.